curriculum/challenges/english/25-front-end-development/quiz-css-variables/66ed9018f45ce3ece4053eb9.md

```diff
@@ -17,439 +17,444 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+If you’ve declared a custom property for animation speed using `@property`, and a user adjusts their settings for animations in their browser, how would that affect your custom property? Would they be able to see the changes, or does it stick to the value you’ve set?
 
 #### --distractors--
 
-Placeholder distractor 1
+The animations will run slower regardless of user settings
 
 ---
 
-Placeholder distractor 2
+The user’s settings override your defined speed
 
 ---
 
-Placeholder distractor 3
+The animations will completely stop if the user disables them
 
 #### --answer--
 
-Placeholder answer
+The user’s preferences will take priority, and they can adjust how the animations play out
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In the following code, which value will color have if --main-color is not defined?
+```
+p {
+  color: var(--main-color, pink);
+}
+```
```

Comment:
````suggestion
In the following code, which value will color have if --main-color is not defined?

```css
p {
  color: var(--main-color, pink);
}
```
````

code fences need to be surrounded by empty lines and they need the language name to know which highlighting to use. please check your other code fences and fix this

Reply:
committed the change you suggested, is it fine now?

Reply:
You successfully fixed this issue, yes